lib/internal/errors.js

```diff
@@ -86,3 +86,5 @@ module.exports = exports = {
 // Note: Please try to keep these in alphabetical order
 E('ERR_ASSERTION', (msg) => msg);
 // Add new errors from here...
+E('ERR_UNK_FILE_OPEN_FLAG', 'Unknown file open flag: %s');
```

Comment:
Hmm, do we have a preference between `util.format` strings and arrow functions + templates? I think the latter should be more performant though(haven't tested) @jasnell